src/Uno.UWP/UI/ColorHelper.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Windows.UI
 {
-    public static partial class ColorHelper
+    public sealed partial class ColorHelper
     {
 		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
 		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
```

Comment:
This is copied generated code :-S

Reply:
Probably should be cleaned up in a separate PR, since the PR is already big, and I think I have more to fix in the UWPSyncGenerator 😄